digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/masstransitstop/oracle/queries.scala

```diff
@@ -237,6 +237,13 @@ object Queries {
     """.as[Int].list
   }
 
+  def getMunicipalitiesWithoutAhvenanmaa: Seq[Int] = {
+    var AhvenanmaaRoadMaintainerId = 0
```

Comment:
That was just to identify that the id of Ahvenanmaa road maintainer is 0. But ok, I can remove it